src/main/java/com/refinedmods/refinedstorage/RSItems.java

```diff
@@ -116,21 +132,37 @@
         createBlockItemFor(RSBlocks.EXTERNAL_STORAGE);
         createBlockItemFor(RSBlocks.IMPORTER);
         createBlockItemFor(RSBlocks.EXPORTER);
-        createBlockItemFor(RSBlocks.NETWORK_RECEIVER);
-        createBlockItemFor(RSBlocks.NETWORK_TRANSMITTER);
-        createBlockItemFor(RSBlocks.RELAY);
-        createBlockItemFor(RSBlocks.DETECTOR);
-        createBlockItemFor(RSBlocks.SECURITY_MANAGER);
         createBlockItemFor(RSBlocks.INTERFACE);
         createBlockItemFor(RSBlocks.FLUID_INTERFACE);
-        createBlockItemFor(RSBlocks.WIRELESS_TRANSMITTER);
         createBlockItemFor(RSBlocks.STORAGE_MONITOR);
         createBlockItemFor(RSBlocks.CONSTRUCTOR);
         createBlockItemFor(RSBlocks.DESTRUCTOR);
-        createBlockItemFor(RSBlocks.DISK_MANIPULATOR);
-        createBlockItemFor(RSBlocks.CRAFTER);
-        createBlockItemFor(RSBlocks.CRAFTER_MANAGER);
-        createBlockItemFor(RSBlocks.CRAFTING_MONITOR);
+
+        RSBlocks.CONTROLLER.forEach((color, block) -> {
+            CONTROLLER.put(color, ITEMS.register(RSBlocks.CONTROLLER.get(color).getId().getPath(), () -> new ControllerBlockItem(RSBlocks.CONTROLLER.get(color).get())));
+            if (color == DyeColor.LIGHT_BLUE) {
+                COLORED_ITEM_TAGS.put(ItemTags.createOptional(new ResourceLocation(RS.ID, block.getId().getPath())), CONTROLLER);
```

Comment:
Why only fill the tag up with the light blue controller type?

Reply:
This creates the tags. They are named after the Block with the lightblue color (the original blocks). 
I'm not actually sure this is needed. I used it for generating the tags.

Reply:
Ah, so by passing `CONTROLLER` you create tags for all the colors. Got it.

Can't this be moved out of the loop? It's a little confusing.